sql/infolinkshell.sql

```diff
@@ -23,7 +23,7 @@ DROP TABLE IF EXISTS `infolinkshell`;
 /*!40101 SET @saved_cs_client     = @@character_set_client */;
 /*!40101 SET character_set_client = utf8 */;
 CREATE TABLE `infolinkshell` (
-  `LinkshellId` int(20) NOT NULL AUTO_INCREMENT,
+  `LinkshellId` bigint(20) NOT NULL AUTO_INCREMENT,
```

Comment:
``` `LinkshellId` bigint(20) UNSIGNED NOT NULL AUTO_INCREMENT,```

though at some point we need to do the same for all ids (e.g. MasterCharacterId on the line below)

Reply:
Keep in mind there is currently a sql rewrite ongoing